src/installer/pkg/projects/netcoreapp/src/netcoreapp.depproj

```diff
@@ -1,7 +1,6 @@
 <Project>
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
   <PropertyGroup>
-    <FrameworkPackageName>Microsoft.NETCore.App</FrameworkPackageName>
```

Comment:
Where does this prop value come from now? (Note `SharedFrameworkName` vs. `FrameworkPackageName` and usages in [framework.dependency.targets](https://github.com/dotnet/arcade/blob/5966c5dd9deb5aca8fd8183eec454cf6cf864d14/src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/framework.dependency.targets))

Reply:
I didn't see that it was being used in arcade, only grepped in dotnet/runtime. Reverting change